jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/ImageEnricher.java

```diff
@@ -83,7 +83,7 @@ public ImageEnricher(JKubeEnricherContext buildContext) {
         PULL_POLICY("pullPolicy");
 
         @Getter
-        protected String key;
+        private String key;
```

Comment:
I'm not sure if this might affect deserialization at some point. Let's just not modify this in this PR.
We can reevaluate and change the rest of Configs.Config accordingly (all of them declare the fields as protected)